include/wsl/winadapter.h

```diff
@@ -314,3 +324,17 @@ extern "C++"
 
 #define IID_PPV_ARGS(ppType) __uuidof(**(ppType)), IID_PPV_ARGS_Helper(ppType)
 #endif
+
+#if defined(lint)
```

Comment:
I'm not familiar with `lint`. Is this the standard way to check for it? I would've expected something like `__LINT__` or `_LINT`.

Reply:
This implementation was lifted directly from the Windows 10 SDK (19041) version of ``ntdef.h``.  The classic "Win32" implementation generates warnings with clang/LLVM for Windows otherwise.